tests/common/test_event.py

```diff
@@ -19,23 +19,50 @@
 
 import json
 import os
-import threading
 from datetime import datetime, timedelta
 
 from azurelinuxagent.common import event, logger
-from azurelinuxagent.common.event import add_event, elapsed_milliseconds, EventLogger, report_metric, WALAEventOperation
+from azurelinuxagent.common.event import add_event, elapsed_milliseconds, EventLogger, report_metric, \
+    WALAEventOperation, parse_xml_event, parse_json_event, parse_event
 from azurelinuxagent.common.exception import EventError
 from azurelinuxagent.common.future import OrderedDict, ustr
 from azurelinuxagent.common.protocol.wire import GoalState
-from azurelinuxagent.common.telemetryevent import TelemetryEventParam
+from azurelinuxagent.common.telemetryevent import TelemetryEvent, TelemetryEventParam
 from azurelinuxagent.common.utils import fileutil
 from azurelinuxagent.common.utils.extensionprocessutil import read_output
-from azurelinuxagent.common.version import CURRENT_AGENT, CURRENT_VERSION
-from azurelinuxagent.ga.monitor import MonitorHandler
-from tests.tools import AgentTestCase, data_dir, load_data, Mock, patch
+from azurelinuxagent.common.version import AGENT_NAME, CURRENT_AGENT, CURRENT_VERSION
+from tests.ga.test_monitor import SysInfoMock
```

Comment:
You should refactor SysInfoMock into its own file.

BTW, currently the definition is on test_sysinfo, not test_monitor :)